Description: Install the PAM module under /usr/lib (merged-usr)
 Debian 13 is merged-usr; lintian rejects files under /lib (aliased-location).

--- a/debian/rules
+++ b/debian/rules
@@ -19,7 +19,7 @@
 	gnulib-tool --makefile-name=Makefile.gnulib --libtool --import fcntl crypto/md5 array-list list xlist getrandom realloc-posix explicit_bzero xalloc getopt-gnu
 
 override_dh_auto_configure:
-	dh_auto_configure -- --enable-pamdir=/lib/$(DEB_HOST_MULTIARCH)/security --docdir=/usr/share/doc/libpam-tacplus
+	dh_auto_configure -- --enable-pamdir=/usr/lib/$(DEB_HOST_MULTIARCH)/security --docdir=/usr/share/doc/libpam-tacplus
 
 override_dh_install:
 	mkdir -p debian/libpam-tacplus/usr/share/pam-configs
--- a/debian/libpam-tacplus.install
+++ b/debian/libpam-tacplus.install
@@ -1,2 +1,2 @@
-lib/*/security/*.so
+usr/lib/*/security/*.so
 usr/share/doc/libpam-tacplus/sample.pam
